src/components/NavBar.js

```diff
@@ -1,16 +1,21 @@
 import React from "react"
+import { connect } from "react-redux"
 import AppBar from "@material-ui/core/AppBar"
 import Toolbar from "@material-ui/core/Toolbar"
 import Button from "@material-ui/core/Button"
 import { AuthButton } from "./Auth"
-// import Typography from '@material-ui/core/Typography';
 import { MemoryRouter as Router, withRouter } from "react-router-dom"
 import redLambdaLogo from "../assets/Lambda_Logo.png"
 
+
 // I've imported Material UI packages above,
 // and implemented them in a functional component below
 
 const NavBar = props => {
+  const { user } = props
+  console.log("user", user)
+  // console.log('user role', user.role)
```

Comment:
Make sure to remove your console logs before you take it off draft stage!

Reply:
thank you! we're still trying to figure this out, i'll remove them before we're ready for review